Example/SwiftUIExample/MobilePaymentsSwiftUIExample/Screens/Home/HomeView.swift

```diff
@@ -0,0 +1,220 @@
+import SquareMobilePaymentsSDK
+import SwiftUI
+
+#if canImport(MockReaderUI)
+import MockReaderUI
+#endif
+
+struct HomeView: View {
+
+    @State private var presentingPermissionsView: Bool = false
+    @State var viewModel: HomeViewModel
+    
+    private let viewHolder: MobilePaymentsSDKViewHolder = MobilePaymentsSDKViewHolder()
+    private var mobilePaymentsSDK: SDKManager { viewModel.mobilePaymentsSDK }
+    private var isIPad: Bool {
+        UIDevice.current.userInterfaceIdiom == .pad
+    }
+
+    init(viewModel: HomeViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        // Some APIs of SquareMobilePaymentsSDK require a UIViewController,
+        // you can use a UIViewControllerRepresentable with an empty UIViewController
+        // and put it on the bottom of a ZStack
+        ZStack {
+            viewHolder
+            VStack {
+                headerView
+                contentView
+                Spacer()
+            }
+            .alert(isPresented: $viewModel.showPaymentStatusAlert) {
+                paymentStatusAlert
+            }
+        }
+        .padding([.leading, .trailing], isIPad ? 50 : nil)
+        .padding([.top, .bottom])
+        .background(Color.white)
+        .onAppear (perform: setupMockReader)
+        .onChange(of: viewModel.authorizationState, perform: { _ in
+            setupMockReader()
+        })
+    }
+    
+    // MARK: - Subviews
+    
+    private var headerView: some View {
+        HStack {
+            settingsButton
+            Spacer()
+            permissionsButton
+        }
+        .padding(.bottom, 50)
+    }
+
+    private var contentView: some View {
+        VStack {
+            Image("donut")
+                .resizable()
+                .aspectRatio(1.0, contentMode: .fit)
+                .frame(width: 248)
```

Comment:
nit: It would be nice to not have a magic number here.

Reply:
Added a private constants enum